.circleci/publish.sh

```diff
@@ -0,0 +1,12 @@
+version=$(cat ../package.json | jq .version | tr -d '"')
+isLatest=`curl -s -X GET \
+"https://api.supertokens.io/0/frontend/latest/check?password=$SUPERTOKENS_API_KEY&version=$version&name=website" \
+-H 'api-version: 0'`
+if [[ `echo $isLatest | jq .isLatest` == "true" ]]
+then
+    cd ..
+    npm publish --tag latest
+else
+    cd ..
+    npm publish --tag version-$version
```

Comment:
Why do we need to prepend with `version-`?

Reply:
a tag name must not be a valid SemVer (Semantic versioning) range (example 2.0.1 or v2.1, etc..)

Reply:
<img width="653" alt="Screenshot 2021-07-22 at 10 41 48 AM" src="https://user-images.githubusercontent.com/21988812/126592582-58dbbff3-bdf0-4760-a37a-de274c5488a2.png">